src/training/params.py

```diff
@@ -311,7 +323,12 @@ def parse_args(args):
         action='store_true',
         help="Freeze BatchNorm running stats in image tower for any locked layers.",
     )
-
+    parser.add_argument(
+        "--context-length",
```

Comment:
This should be configured at the model type level rather than being a param

Reply:
I agree it's better to consolidate it to model configs, but currently context_length is not reflected to the tokenizer.
Perhaps, it's a separate work item for corpora with longer text.

Reply:
@usuyama I made an issue for this (https://github.com/mlfoundations/open_clip/issues/222). Let's discuss how to best do it there